coin_cbc_sys/src/lib.rs

```diff
@@ -22,7 +22,11 @@ pub type cbc_callback = Option<
     ),
 >;
 
-#[link(name = "CbcSolver")]
+#[cfg_attr(
+    target_family = "windows",
+    link(name = "libCbcSolver", kind = "static")
```

Comment:
Seems to still work fine if you drop `kind = "static"`; should it really be forced?